src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx

```diff
@@ -39,7 +42,10 @@ const MultiAccordionWithTitle: FC<MultiAccordionWithTitleProps> = ({
   }
 
   const accordionTitle = title && (
-    <Typography sx={sx.title} variant={VariantEnum.H5}>
+    <Typography
+      sx={{ ...sx.title, ...(isMobile && { m: '16px 20px 10px' }) }}
+      variant={isMobile ? VariantEnum.H6 : VariantEnum.H5}

```

Comment:
I'm not sure that it needs here. 
...(isMobile && { m: '16px 20px 10px' })

Reply:
removed